### ISSUE TITLE
INSTALL/MYSQL.sql is broken

### ISSUE THREAD

**Issue by Rafiot** [MISP/MISP#764]
```
$ mysql -u misp -pblah misp < INSTALL/MYSQL.sql
ERROR 1060 (42S21) at line 250: Duplicate column name 'name'
The command "mysql -u misp -pblah misp < INSTALL/MYSQL.sql" failed and exited with 1 during .
```

**Comment by Rafiot** [comment]
Still broken.

```
ERROR 1064 (42000) at line 647: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ''temp_template', 'perm_sharing_group')

VALUES ('1', 'admin', NOW() , NOW() , '1'' at line 1

The command "mysql -u misp -pblah misp < INSTALL/MYSQL.sql" failed and exited with 1 during .
```

**Comment by Rafiot** [comment]
WIP: https://github.com/Rafiot/MISP/commit/459507e019d4102802bcd205e601b932d3e9972f

**Comment by adulau** [comment]
Fixed in 243e6284f171823aed096877ce5604b8c12aad2b
